app.py

```diff
@@ -73,3 +73,10 @@ def index():
     action.perform_action()
 
     return ''
+
+@app.route('/event', methods=['POST'], content_types=['application/x-www-form-urlencoded'])
+def event():
+    req = app.current_request.raw_body.decode()
+    payload = slack_payload_extractor(req)
+    logger.info(f'payload is: {payload}')
+    return payload.get('challenge')
```

Comment:
Why do we return the challenge? Is this safe to do or could it be used to do something that we don't by someone else?

Reply:
thumbs down means no 😂 Im on the phone

Reply:
Since we don't verify slack token, doesn't this mean if someone would know this endpoint, they could exploit this bot to do all sorts of things?